test/models/devolved_administration_availability_test.rb

```diff
@@ -9,4 +9,55 @@ class DevolvedAdministrationAvailabilityTest < ActiveSupport::TestCase
     assert_equal "local_authority_service", devolved_administration_availability.type
     assert_equal "", devolved_administration_availability.alternative_url
   end
+
+  test "should not be valid with an unallowed type value" do
+    devolved_administration_availability = FactoryBot.build(
+      :devolved_administration_availability,
+      type: "invalid_type",
+    )
+    assert_not devolved_administration_availability.valid?
+  end
+
+  test "should be valid with an allowed type value" do
+    devolved_administration_availability = FactoryBot.build(
+      :devolved_administration_availability,
```

Comment:
It'd be good if this test had a type argument provided as otherwise it looks like it's testing the factory.

Reply:
Thanks. I updated!